soccer/gameplay/constants.py

```diff
@@ -73,8 +74,10 @@ class Field:
 
     TheirGoalSegment = robocup.Segment(robocup.Point(GoalWidth / 2.0, Length),
                                         robocup.Point(-GoalWidth / 2.0, Length))
+    TheirGoalCenter = robocup.Point(0 , Length)
```

Comment:
Let's remove this and the OurGoalCenter value below.  The Segment class has a [center() method](https://github.com/robojackets/robocup-software/blob/master/soccer/gameplay/robocup-py.cpp#L302), so it doesn't add much.  Also, if we wanted to have these, we'd have to modify the `setFieldConstantsFromField_Dimensions()` method below to update their values if we switch field sizes.